docs/deploy.zh-CN.md

```diff
@@ -37,53 +39,33 @@ $ cd apisix-dashboard
 
 2. 检查环境变量
 
-- 开启 Go MODULE
-
-```sh
-$ go env -w GO111MODULE=on
-```
-
 - 对于大多数中国用户，我们可以使用 [Goproxy](https://goproxy.cn/) 加快模块下载速度。
 
 ```sh
 $ go env -w GOPROXY=https://goproxy.cn,direct
 ```
 
-3. 构建
-
-```sh
-$ api/build.sh
-```
-
-构建完成后的文件在根目录 `/output` 下。
-
-## 构建前端
+### 前端 检查项
 
 该项目使用 [Ant Design Pro](https://pro.ant.design) 初始化。以下是一些使用方法的快速指南。
 
 1. 确保你的设备已经安装了 `Node.js(版本 10.0.0+)`。
 
 2. 安装 [yarn](https://yarnpkg.com/)。
 
-3. 安装依赖:
 
-```sh
-$ cd /web
+### 开始构建
 
-$ yarn install
-```
-
-4. 构建
 
 ```sh
-$ yarn build
+$ make dashboard-build
 ```
 
-构建完成后的文件在根目录 `/output/html` 目录下。
+构建完成后的文件在根目录 `/output` 下。
 
 ## 启动
 
-1. 根据您的部署环境，检查并修改 `api/conf/conf.yaml` 中的配置。
+1. 根据您的部署环境，检查并修改 `output/conf/conf.yaml` 中的配置。
```

Comment:
we can remove the `output/` here.

Reply:
It will be done with the job of adjust directory structure #731